raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -208,8 +193,8 @@ public KafkaRaftClient(NetworkChannel channel,
                            int fetchMaxWaitMs,
                            LogContext logContext,
                            Random random) {
+        // FIXME: Validate voter connections against stored cache state
```

Comment:
Since the KIP is approved, we could get AK tickets to track this.

Reply:
I'll remove this. Most of the TODOs got addressed in #402 anyway.